### ISSUE TITLE
Review rating missing meaningful labels

### ISSUE THREAD

**Issue by ravenokko** [shopware/shopware#13219]
### Shopware Version

6.7.3.1

### Affected area / extension

Platform(Default)

### Actual behaviour

Review rating does not have meaningfully associated labels, tested in Safari. VoiceOver merely tells you the number of the star, not the contextual label of the rating scale (for example: people might think 2 out of 5 stars is Unsatisfactory).  While VoiceOver can eventually reach the text, it puts an additional mental burden on screen reader users to make the connection.  These are WCAG failures per [F96](https://www.w3.org/WAI/WCAG22/Techniques/failures/F96) and [F111](https://www.w3.org/WAI/WCAG22/Techniques/failures/F111)

https://github.com/user-attachments/assets/00655562-c794-4391-bac3-78a67dcd3d0d

### Expected behaviour

- When selecting a star, VoiceOver also reads 'Unsatisfactory/Acceptable/Good/Very Good/Excellent'
- ProductReview.spec.ts can then utilize the [Actor.selectsRadioButton()](https://github.com/shopware/acceptance-test-suite/blob/01dfa05b3b3cc92761e721c31227b46733a8b466/src/services/Actor.ts#L59) functionality.

Here is a nearly identical example of how to do so: https://www.w3.org/WAI/tutorials/forms/custom-controls/#a-star-rating

### How to reproduce

Write a review using VoiceOver in Safari
